libsignal-service/src/provisioning/manager.rs

```diff
@@ -248,6 +250,7 @@ impl<P: PushService> LinkingManager<P> {
             .await?;
 
         let registration_id = csprng.gen_range(1, 256);
+        let pni_registration_id = csprng.gen_range(1, 256);
```

Comment:
I actually wanted to test it but there's a method we use from `libsignal` that does this in a wider range, and I think it's like this because it's inconsistent (or old). But maybe we can change this in another PR.

Reply:
Yeh, I think it needs to be until somewhere in the 16k...